### DISCUSSION TITLE
Properly defined URLs returning 404

### DISCUSSION THREAD

**Discussion by tusharhsim** [Healthlane-Technologies/Zango#277] [Q&A] [answered]
hi, while trying to setup the project in windows, i'm unable to visit defined endpoints. below is the workspace folder structure:
![image](https://github.com/Healthlane-Technologies/Zango/assets/43616557/440d1e3a-fc0d-4e21-9c20-207c6be09742)
even after defining settings.json for the test_app and simple view, not able to access it in browser, the screenshots:
![image](https://github.com/Healthlane-Technologies/Zango/assets/43616557/bad97946-fee6-4220-9097-dd292b1859dc)
![image](https://github.com/Healthlane-Technologies/Zango/assets/43616557/a3a09100-d4d4-4bdb-9ad8-846eef557728)
![image](https://github.com/Healthlane-Technologies/Zango/assets/43616557/6e2d2413-041a-43bd-9db2-86b80f557a3a)
getting below response:
![image](https://github.com/Healthlane-Technologies/Zango/assets/43616557/bdb9e9de-9d18-4f4c-a530-c5b685b8fb50)

can you let me know if i'm missing something?
also i duplicated steps mentioned in https://github.com/Healthlane-Technologies/Zango/tree/main/examples/tutorial_app, still the same issue

**Comment by kc-diabeat** [comment, marked as answer]
Hi @tusharhsim, Zango is a multi-tenant framework. Application are identified with unique domain_url's. It seems you are trying to access the application at localhost which is incorrect. You will have to assign a domain for the application from the app panel and make sure the domain is mapped to localhost. You can do this by adding an entry in the /etc/hosts file on your windows machine.

**Reply by tusharhsim** [reply]
hi @kc-diabeat, thanks for quick response, after suggested configurations the app is running fine. thanks!
